src/languages/typescript.ts

```diff
@@ -188,6 +188,11 @@ const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
   regularExpression: "regex",
   className: ["class_declaration[name]", "class[name]"],
   functionCall: ["call_expression", "new_expression"],
+  functionCallee: [
+    "call_expression.identifier!",
+    "call_expression.call_expression!",
+    "new_expression.identifier!",
```

Comment:
as above
```suggestion
    "new_expression[constructor]",
```

(and tests for corner cases, eg `new (foo[0])()`)

Reply:
It looks like you have one extra closing paren there. Also do you want it to select the new as well? It selects correctly other than that.